src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,155 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceTest {
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void setupRegistrationService() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void registration_validUser_ok() {
+        user = new User();
+        user.setLogin("QAValid1");
+        user.setPassword("12345667");
+        user.setAge(22);
+        user.setId(1L);
+        User registrationResult = registrationService.register(user);
```

Comment:
```suggestion
        user.setAge(22);
        User registrationResult = registrationService.register(user);
```

we dont need to set id